AreaCalculations/AreaCalculations/Calculations.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AreaCalculations
+{
+    class Calculations
+    {
+        Square square = new Square();
+        Rectangle rectangle = new Rectangle();
+        public enum Figures
+        {
+            Square = 1,
+            Rectangle
+        }
+        public Figures GetFigureType(string figure)
+        {
+            Figures result = 0;
+            switch (figure)
+            {
+                case "Квадрат":
+                    result = Figures.Square;
+                    break;
+                case "Прямоугольник":
+                    result = Figures.Rectangle;
+                    break;
+            }
+            return result;
+        }
+        public float GetResult(Figures figures)
```

Comment:
This method should be reworked. 
We will discuss it later.
If you have time investigate creation patterns in c# (e.g. pattern Builder - what is it, what are its purposes, how to use it, etc.)